python/cudf/cudf/core/column/string.py

```diff
@@ -1109,3 +1506,32 @@ def _string_column_binop(lhs, rhs, op):
     _ = libcudf.binops.apply_op(lhs=lhs, rhs=rhs, out=out, op=op)
     nvtx_range_pop()
     return out
+
+
+def _get_cols_list(others):
+    from cudf.core import Series, Index
+    from cudf.core.column import as_column
+
+    if (
+        is_list_like(others)
+        and others
```

Comment:
Just to be more explicit for easier following:
```suggestion
        and len(others) > 0
```